configs/components/cpp-pcp-client.rb

```diff
@@ -69,6 +69,14 @@
     ["#{make} -j$(shell expr $(shell #{platform[:num_cores]}) + 1)"]
   end
 
+  # Can't run binaries when cross-compiling, and Solaris has an issue with
+  # accessing the certs while testing. AIX tests are always iffy.
+  if !platform.is_cross_compiled? && !platform.is_solaris? && !platform.is_aix?
```

Comment:
I'm actually not sure, I just recall them being trouble on Facter. I can take a look at whether they should work for these.